### ISSUE TITLE
Alt text is not descriptive [push-notification pages]

### ISSUE THREAD

**Issue by timngyn** [aws-amplify/docs#7064]
**Describe the content issue**:
Images listed below do not have descriptive/informative alt text. For markdown the alt text is located between the square brackets, `![alt text](...)`.

The list below isn't exhaustive since I found these images based on the word count for the alt text, but it should be a good starting point to make our alt text better for people who use screen readers.

Some resources to help write alt text:
- https://webaim.org/techniques/alttext/
- https://www.a11yproject.com/posts/are-you-making-these-five-mistakes-when-writing-alt-text/

&nbsp;

https://github.com/aws-amplify/docs/blob/main/src/fragments/lib/push-notifications/android/setup_push_service/setup-fcm.mdx:
```
  14  
  15: ![image](/images/push-notifications/setup-fcm/project-settings.png)
  16  

  19  
  20: ![image](/images/push-notifications/setup-fcm/manage-api.png)
  21  

  25  
  26: ![image](/images/push-notifications/setup-fcm/server-id.png)
  27  
```

&nbsp;

https://github.com/aws-amplify/docs/blob/main/src/fragments/lib/push-notifications/common/enable-rich-notifications.mdx:
```
  43  
  44: ![Extension info](/images/push-notifications/cross-platform-service-extension/04_extension-info.png)
  45  

  47  
  48: ![Extension class](/images/push-notifications/cross-platform-service-extension/05_extension-class.png)
```

&nbsp;

https://github.com/aws-amplify/docs/blob/main/src/fragments/lib/push-notifications/ios/getting_started/20_set_entitlements.mdx:
```
  22  
  23: ![Add capability](/images/push-notifications/xcode-entitlements/01_add-capability.png)
  24  

  32  
  33: ![Add push](/images/push-notifications/xcode-entitlements/02_add-push.png)
  34  

  36  
  37: ![App sandbox](/images/push-notifications/xcode-entitlements/05_app-sandbox.png)
  38  

  40  
  41: ![Outgoing connections](/images/push-notifications/xcode-entitlements/06_app-sandbox-outgoing-connections.png)
  42  
```

&nbsp;

https://github.com/aws-amplify/docs/blob/main/src/fragments/lib/push-notifications/ios/remote_media/10_remote_media.mdx:
```
  12  
  13: ![New target](/images/push-notifications/service-extension/01_new-target.png)
  14  

  16  
  17: ![Target type](/images/push-notifications/service-extension/02_target-type.png)
  18  

  20  
  21: ![Target name](/images/push-notifications/service-extension/03_target-name.png)
  22  

  24  
  25: ![Add library](/images/push-notifications/service-extension/04_add-library.png)
  26  

  28  
  29: ![Notification Service](/images/push-notifications/service-extension/05_notification-service.png)
  30  
```

&nbsp;

https://github.com/aws-amplify/docs/blob/main/src/fragments/lib/push-notifications/ios/setup_push_service/setup-apns.mdx:
```
  35  
  36: ![image](/images/push-notifications/setup-apns/create-identifier--add.png)
  37  

  39  
  40: ![image](/images/push-notifications/setup-apns/create-identifier--app-id.png)
  41  

  43  
  44: ![image](/images/push-notifications/setup-apns/create-identifier--app-type.png)
  45  

  48  
  49: ![image](/images/push-notifications/setup-apns/create-identifier--bundle-id.png)
  50  

  52  
  53: ![image](/images/push-notifications/setup-apns/create-identifier--enable-PN.png)
  54  

  69  
  70: ![image](/images/push-notifications/setup-apns/create-certificate--configure.png)
  71  

  73  
  74: ![image](/images/push-notifications/setup-apns/create-certificate--create.png)
  75  

  77  
  78: ![image](/images/push-notifications/setup-apns/create-certificate--cert-assistant.png)
  79  

  81  
  82: ![image](/images/push-notifications/setup-apns/create-certificate--upload-cert.png)
  83  

  88  
  89: ![image](/images/push-notifications/setup-apns/create-certificate--my-cert.png)
  90  

  92  
  93: ![image](/images/push-notifications/setup-apns/create-certificate--export-cert.png)
  94  
```

&nbsp;

https://github.com/aws-amplify/docs/blob/main/src/fragments/lib-v1/push-notifications/react-native/enable_rich_notifications/enable-rich-notifications.mdx:
```
  37  
  38: ![Extension info](/images/push-notifications/cross-platform-service-extension/04_extension-info.png)
  39  

  41  
  42: ![Extension class](/images/push-notifications/cross-platform-service-extension/05_extension-class.png)
```
